drizztsaga/scripts/baf/bg/f_drizzt.baf

```diff
@@ -21,18 +56,21 @@ END
 
 // to prevent immersion-breaking multiple instances of Drizzt's scimitars (if pickpocketed drizzt before he joins)
 IF
-  Delay(10)
-  GlobalLT("F_DrizztVanillaStuffDisposed","LOCALS",3)
+  Delay(1)
+  GlobalLT("F_DrizztVanillaStuffDisposed","LOCALS",4)
```

Comment:
same question here

Reply:
It is not optimun for sure, will use GT instead.
For the record this block was execute only once but it need to trigger with each OR conditions...

Reply:
I mean, if this script block sets the variable to =3, then this condition still could apply because it is still LT 4

Reply:
yes that the the issue (edge case I assume, but you never know)

Reply:
In my understanding, `GlobalLT("F_DrizztVanillaStuffDisposed","LOCALS",3)` should be correct condition. The script block sets var to 3, so it IS lower than 3 in the condition. What do you think about reverting this line?